transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringEventLoop.java

```diff
@@ -109,6 +108,7 @@ void add(AbstractIOUringChannel ch) {
 
     void remove(AbstractIOUringChannel ch) {
         logger.trace("Remove Channel: {}", ch.socket.intValue());
+        channels.remove(ch.socket.intValue());
```

Comment:
remove this change as we already call remove below

Reply:
yeah my bad